Gemfile

```diff
@@ -88,8 +87,6 @@ gem "wim_parser",                       "~>1.0",             :require => false
 gem "cgi",  "~> 0.3.5"
 # CVE-2023-28756 fixed: ruby 3.1.4 - https://github.com/advisories/GHSA-fg7x-g82r-94qc
 gem "time", "~> 0.2.2"
-# CVE-2023-36617 https://github.com/advisories/GHSA-hww2-5g85-429m
-gem "uri",  "~> 0.10.3"
```

Comment:
Is this ok for ruby 3.0 @kbrock ?

Reply:
@jrafanie this is an issue for 3.1.4 (rpm version currently shipped) and 3.1.6 (latest version of ruby).

I am not aware of any version of ruby that solves this issue.

Please keep it in.

Reply:
The problem is I can't bundle on ruby 3.1 with it

Reply:
It would be unfortunate, but could you wrap this in a ruby version check?

Reply:
I changed it to >= `0.10.3`

Reply:
ugh, I must have put in the wrong version then

```
gem "uri", "> 0.12.2"
```

Reply:
ok, that works on 3.0 and 3.1...

Reply:
haha, it's 0.12.2+ for 3.1+ and 0.10.3+ for 3.0 according to https://github.com/advisories/GHSA-hww2-5g85-429m

Reply:
Ok, I think this question is resolved.